model/driver/src/icon4py/model/driver/test_cases/gauss3d.py

```diff
@@ -11,6 +11,7 @@
 
 import gt4py.next as gtx
 
+import icon4py.model.common.grid.geometry
```

Comment:
```suggestion
import icon4py.model.common.grid.geometry as geometry
```

Reply:
that does not really add anything. You only need the rename if you want to rename...

Reply:
fixed it